pinot-core/src/test/java/org/apache/pinot/queries/TextSearchQueriesTest.java

```diff
@@ -1864,6 +1878,26 @@ public void testInterSegment() {
     query = "SELECT count(*) FROM MyTable WHERE TEXT_MATCH(SKILLS_TEXT_COL_DICT, 'a and or in the are')";
     testInterSegmentAggregationQueryHelper(query, 0);
 
+    // query with excluded stop-words. they should be indexed
```

Comment:
This seems a bit confusing. If this test is for excluding the stop words, then the comment should say `"they should not be indexed"` I guess ?

Reply:
Nvm.. I think I got it. 

I think what you mean is `"exclude the stop words from stop word list"` and thus they should be indexed

Reply:
Updated the comment